backend/lib/cache/index.js

```diff
@@ -48,25 +77,34 @@ module.exports = {
     return cache.getSeeds()
   },
   getSeed (name) {
-    return _.cloneDeep(_.find(cache.getSeeds(), ['metadata.name', name]))
+    return _
+      .chain(cache.getSeeds())
+      .find(['metadata.name', name])
+      .cloneDeep()
+      .value()
   },
   getVisibleAndNotProtectedSeeds () {
     const predicate = item => {
-      const taints = _.get(item, 'spec.taints', [])
-      let seedProtected = false
-      let seedInVisible = false
-      _.forEach(taints, taint => {
-        if (taint.key === 'seed.gardener.cloud/protected') {
-          seedProtected = true
-        }
-        if (taint.key === 'seed.gardener.cloud/invisible') {
-          seedInVisible = true
-        }
-      })
-      return !seedProtected && !seedInVisible
+      const taints = _.get(item, 'spec.taints')
+      const unprotected = !_.find(taints, ['key', 'seed.gardener.cloud/protected'])
+      const visible = !_.find(taints, ['key', 'seed.gardener.cloud/invisible'])
+      return unprotected && visible
     }
     return _.filter(cache.getSeeds(), predicate)
   },
+  getProjects () {
+    return cache.getProjects()
```

Comment:
check if cache is stale (for all getters)

Reply:
✔️